code/ATMOSPHERICS/components/unary_devices/outlet_injector.dm

```diff
@@ -177,6 +232,14 @@
 "}
 
 /obj/machinery/atmospherics/unary/outlet_injector/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/pen))
+		var/t = copytext(stripped_input(user, "Enter the name for the injector.", "Rename", name), 1, MAX_NAME_LEN)
+		if(!t)
+			return
+		if(!in_range(src, usr) && loc != usr)
```

Comment:
use `user` instead of `usr` here. Rule of thumb is to use `usr` if user is not given to you in the params. And if it's not given to you in the params then try to figure out why and maybe change it if needed.